app/controllers/cycle_controller.rb

```diff
@@ -0,0 +1,7 @@
+class CycleController < ApplicationController
```

Comment:
this file should be called `cycles_controller.rb` and the class should be `CyclesController`